tests/playwright-test/golden.spec.ts

```diff
@@ -65,6 +65,77 @@ Line7`,
   expect(result.output).toContain('Line7');
 });
 
+test('should write detailed failure result to an output folder', async ({runInlineTest}, testInfo) => {
+  const result = await runInlineTest({
+    'a.spec.js-snapshots/snapshot.txt': `Hello world`,
+    'a.spec.js': `
+      const { test } = pwt;
+      test('is a test', ({}) => {
+        expect('Hello world updated').toMatchSnapshot('snapshot.txt');
+      });
+    `
+  });
+
+  expect(result.exitCode).toBe(1);
+  expect(result.output).toContain('Snapshot comparison failed:');
+  const expectedSnapshotArtifactPath = testInfo.outputPath('test-results', 'a-is-a-test', 'snapshot-expected.txt');
+  const actualSnapshotArtifactPath = testInfo.outputPath('test-results', 'a-is-a-test', 'snapshot-actual.txt');
+  expect(result.output).toMatch(new RegExp(` {4}Expected: .+?${expectedSnapshotArtifactPath}`));
```

Comment:
This one fails on Windows, probably because '\\' in RegExp has a special meaning. I'd suggest to use:
```
expect(result.output).toContain(`Expected: ${expectedSnapshotArtifactPath}`)
```

Reply:
Because the path is colored it's wrapped in some invisible chars and toContain doesn't work the way you've written. I have 4 options what to do:  
- Test that path exists without describtion `expect(result.output).toContain(expectedSnapshotArtifactPath)`
- Escape path in RegExp using `.replace(/[#$()*+,.?[\\\]^{|}-]/g, '\\$&')`
- Import `colors` and wrap the path with `colors.yellow` in test
- Not to test  the output result  

What do you think?

Reply:
I think you can use `stripAscii` function we use in other places: https://github.com/microsoft/playwright/blob/742cce3a1d578dd736b3f949d22efa4794fc064e/tests/playwright-test/base-reporter.spec.ts#L126. If that does not work, just remove this `expect` call, it's not worth too much trouble.